### DISCUSSION TITLE
Sitios de inspiración ¿? :P

### DISCUSSION THREAD

**Discussion by mabvmex** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#324] [Q&A]
¡Hola a todos!

Viendo las prácticas de la semana 3, me gustaría conocer sitios donde pueda ver diseños de sitios web que yo pueda replicar;
en los videos esta semana se habló un poco de behance y dribbble...

¿Ustedes conocen otros sitios?

Saludos.

**Comment by doguedogue** [comment]
la clave es buscar PLANTILLAS o TEMPLATES, con las herramientas que tenemos hasta ahora se puede replicar casi todo, mira que muchas las venden.

https://nicepage.com/es/plantillas-html

**Reply by jmlimas** [reply]
Gracias por  compartir doguedogue

**Comment by DanyVeneno** [comment]
Que tal mabvmex yo te dejo unos 
1-https://www.cssdesignawards.com/
2-https://www.awwwards.com/
3-https://www.csswinner.com/
estos son inspiradores

Metete tambien a Pinterest ahi hay cosas muy chidas igual
